src/frontend/src/utils/helpers.ts

```diff
@@ -118,3 +119,77 @@ export function getClientTimezone() {
   }
   return "Etc/UTC";
 }
+
+/**
+ * Helper function to auto-paginate queries for an nodeinfra endpoint
```

Comment:
```suggestion
Helper function to auto-paginate queries for a DSS REST API endpoint
```

Note that the reference frontend example config includes Nodeinfra but that anyone's DSS can be used